src/panels/config/backup/dialogs/show-dialog-download-decrypted-backup.ts

```diff
@@ -0,0 +1,40 @@
+import { fireEvent } from "../../../../common/dom/fire_event";
+import type { BackupContent } from "../../../../data/backup";
+
+export interface DownloadDecryptedBackupDialogParams {
+  backup: BackupContent;
+  agentId?: string;
+  submit?: () => void;
+  cancel?: () => void;
```

Comment:
These are not used, and we never call submit, only always cancel, should we just remove them?